src/jobs/go-lint.yaml

```diff
@@ -0,0 +1,47 @@
+description: "Runs golanglint-ci against the codebase."
+parameters:
+  disable:
+    default: ""
+    description: "The names of default linting modules to skip."
+    type: "string"
+  disable-all:
+    default: false
+    description: "If set, will disable all linters (except those included by '--enable')."
+    type: "boolean"
+  enable:
+    default: ""
+    description: "The names of additional linting modules to run."
+    type: "string"
+  new-from-rev:
+    default: ""
+    description: "Show only new issues created after git revision REV."
+    type: "string"
+  presets:
+    default: "none"
+    description: "Enable preset list of linters (bugs|complexity|format|performance|style|unused)."
+    type: "enum"
+    enum: ["bugs", "complexity", "format", "performance", "style", "unused", "none"]
+  skip-dirs:
+    default: ""
+    description: "Regular expressions for folder paths which should be skipped."
+    type: "string"
+  skip-files:
+    default: ""
```

Comment:
Why not have `vendor` here?

Reply:
It is already the default

Reply:
Okay I get it now - the naming `default` is just very confusing to me in that sense :D

Reply:
Yes there are actually Job, Command, and underlying tool defaults to speak of :see_no_evil: